src/main/java/com/epam/aidial/core/service/ResourceService.java

```diff
@@ -740,6 +757,19 @@ public String getEtag(ResourceDescription descriptor) {
         return metadata.getEtag();
     }
 
+    private static Map<String, String> toUserMetadata(String etag, Long createdAt, Long updatedAt) {
+        Map<String, String> metadata = new HashMap<>();
+        metadata.put(ResourceUtil.ETAG_ATTRIBUTE, etag);
+        if (createdAt != null) {
+            metadata.put(ResourceUtil.CREATED_AT_ATTRIBUTE, Long.toString(createdAt));
```

Comment:
Seems like they are always non nullable.

Reply:
I can't guarantee it. The 'updatedAt' field seems unlikely to be null, but 'createdAt' may be null. For instance, 'createdAt' is not returned by the local filesystem blob store, and possibly by some other blob stores as well.